src/services/doppler-legacy-client.ts

```diff
@@ -397,6 +399,16 @@ function parsePlan(json: any) {
   }
 }
 
+function mapIdUserToken(jwtToken: string) {
+  var idUser = 0;
+  if (jwtToken) {
+    var tokenDecoded = jwt_decode<any>(jwtToken);
+    idUser = tokenDecoded.nameid || 0;
+  }
+
+  return idUser;
```

Comment:
What do you you think about removing the mutation of `idUser` variable in order to get better static analysis? 

```suggestion
  if (jwtToken) {
    var tokenDecoded = jwt_decode<any>(jwtToken);
    return tokenDecoded.nameid || 0;
  }

  return 0;
```

Reply:
👍